awx/main/tests/functional/analytics/test_collectors.py

```diff
@@ -72,6 +73,29 @@ def write_stdout(self, sql, fd):
     return path
 
 
+@pytest.mark.django_db
```

Comment:
I'm still working on the unit tests, but want to get several of these in so that we can make sure we're collecting job events the way we expect.